src/io/github/sspanak/tt9/languages/definitions/Polish.java

```diff
@@ -0,0 +1,22 @@
+package io.github.sspanak.tt9.languages.definitions;
+
+import java.util.Arrays;
+import java.util.Locale;
+
+public class Polish extends English {
+	public Polish() {
+		super();
+
+		locale = Locale.POLISH;
```

Comment:
I am not sure this is a valid locale in Java. Please, make sure to correct everything that causes the build to fail.